README.rst

```diff
@@ -1874,823 +67,14 @@ as in the example below:
-        get_orientdb_schema_graph
-    )
-    from graphql_compiler.schema_generation.orientdb.utils import (
-        ORIENTDB_INDEX_RECORDS_QUERY, ORIENTDB_SCHEMA_RECORDS_QUERY
-    )
-
-    # Get schema metadata from hypothetical Animals database.
-    client = your_function_that_returns_an_orientdb_client()
-    schema_records = client.command(ORIENTDB_SCHEMA_RECORDS_QUERY)
-    schema_data = [record.oRecordData for record in schema_records]
-
-    # Get index data.
-    index_records = client.command(ORIENTDB_INDEX_RECORDS_QUERY)
-    index_query_data = [record.oRecordData for record in index_records]
-
-    # Build SchemaGraph.
-    schema_graph = get_orientdb_schema_graph(schema_data, index_query_data)
-
-    # Get all the subclasses of a class.
-    print(schema_graph.get_subclass_set('Animal'))
-    # {'Animal', 'Dog'}
-
-    # Get all the outgoing edge classes of a vertex class.
-    print(schema_graph.get_vertex_schema_element_or_raise('Animal').out_connections)
-    # {'Animal_Eats', 'Animal_FedAt', 'Animal_LivesIn'}
-
-    # Get the vertex classes allowed as the destination vertex of an edge class.
-    print(schema_graph.get_edge_schema_element_or_raise('Animal_Eats').out_connections)
-    # {'Fruit', 'Food'}
-
-    # Get the superclass of all classes allowed as the destination vertex of an edge class.
-    print(schema_graph.get_edge_schema_element_or_raise('Animal_Eats').base_out_connection)
-    # Food
-
-    # Get the unique indexes defined on a class.
-    print(schema_graph.get_unique_indexes_for_class('Animal'))
-    # [IndexDefinition(name='uuid', 'base_classname'='Animal', fields={'uuid'}, unique=True, ordered=False, ignore_nulls=False)]
-
-In the future, we plan to add :code:`SchemaGraph` generation from SQLAlchemy
-metadata. We also plan to add a mechanism where one can query a
-:code:`SchemaGraph` using GraphQL queries.
-
-Cypher query parameters
-~~~~~~~~~~~~~~~~~~~~~~~
-
-RedisGraph `doesn't support query
-parameters <https://github.com/RedisGraph/RedisGraph/issues/544#issuecomment-507963576>`__,
-so we perform manual parameter interpolation in the
-:code:`graphql_to_redisgraph_cypher` function. However, for Neo4j, we can
-use Neo4j's client to do parameter interpolation on its own so that we
-don't reinvent the wheel.
-
-The function :code:`insert_arguments_into_query` does so based on the query
-language, which isn't fine-grained enough here-- for Cypher backends, we
-only want to insert parameters if the backend is RedisGraph, but not if
-it's Neo4j.
-
-Instead, the correct approach for Neo4j Cypher is as follows, given a
-Neo4j Python client called :code:`neo4j_client`:
-
-.. code:: python
-
-    common_schema_info = CommonSchemaInfo(schema, type_equivalence_hints)
-    compilation_result = compile_graphql_to_cypher(common_schema_info, graphql_query)
-    with neo4j_client.driver.session() as session:
-        result = session.run(compilation_result.query, parameters)
-
-FAQ
----
```

Comment:
This section is essentially copy pasted except for some minor nits/improvements in ReadTheDocs.